easybuild/easyblocks/generic/pythonbundle.py

```diff
@@ -181,6 +181,8 @@ def load_module(self, *args, **kwargs):
         # Required here to ensure that it is defined for sanity check commands of the bundle
         # because the environment is reset to the initial environment right before loading the module
         env.setvar('PYTHONNOUSERSITE', '1', verbose=False)
+        # Set (again) in case the module changes it
+        env.setvar('PIP_REQUIRE_VIRTUALENV', 'false', verbose=False)
```

Comment:
@Flamefire Shouldn't we introduce a function that can be called from both `pythonbundle` and `pythonpackage`?

Reply:
You mean one that sets `PYTHONNOUSERSITE`, `PIP_REQUIRE_VIRTUALENV` and `PIP_DISABLE_PIP_VERSION_CHECK`?

Given that we already need it in quite some places it makes sense, yes. Maybe named `setup_pip_env` with a `verbose` param? I'd make that default to `False` and only set it to `True` in `PythonPackage.__init__` to reduce log spam. Even there it gets likely duplicated a lot for bundles but completely omitting it feels wrong.

Reply:
Sounds good

Reply:
Ok, implemented it in the Python EasyBlock file to avoid cyclic includes of that and PythonPackage. Those variables are also required/useful for TensorFlow & friends, hence I made them a constant.

To avoid the log spam the actual set is only executed if any of the variables need changing.